drivers/serial/uart_stm32.c

```diff
@@ -670,6 +672,52 @@ static int uart_stm32_init(struct device *dev)
 		return -EIO;
 	}
 
+	/* Configure dt provided device signals when available */
+	if (config->pinctrl_list_size != 0) {
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32f1_pinctrl)
```

Comment:
some thing here

Reply:
Same answer :-).
Some of the functions below are only available for stm32f1 series (the remap functions)